README.md

```diff
@@ -26,10 +26,4 @@ Coming soon (Make an issue if you want this really quick)
 
 To only download the materials for a particular course, say COMP40009 Computing Practical, run `scientia-cli download 40009`.
 
-## How to install Go
-1. Get the latest tar for your OS from [here](https://go.dev/dl/) by running `wget <link you just copied>`
-2. Run `rm -rf /usr/local/go && tar -C /usr/local -xzf <go1.XX.X.os-arch.tar.gz>`
-3. Add go to your PATH by running `echo "export PATH=\$PATH:/usr/local/go/bin" >> ~/.bashrc`
-4. Run `source ~/.bashrc` to update your PATH
```

Comment:
They may be using .zshrc or .bashprofile. Better to leave it generic or link to a site which explains how to add to PATH.

Reply:
The `linux-installer.sh` print out how to add it to your PATH. Looks like this

```
======================================================================================================
Add scientia-cli to your path. You can do so by running the command depending upon which shell you use

# BASH
echo "export PATH=\$PATH:/usr/local/bin/scientia-cli" >> ~/.bashrc

# ZSH 
echo "export PATH=\$PATH:/usr/local/bin/scientia-cli" >> ~/.zshrc

# FISH 
set -U fish_user_paths /usr/local/go/bin $fish_user_paths

Run source ~/.bashrc afterwards
======================================================================================================
```

Reply:
At the end, you say `source ~/.bashrc`. Maybe don't include that or make it generic?